blocks.md

```diff
@@ -0,0 +1,111 @@
+<h2>Guide to submitting plugins to the Block Directory</h2>
+
+The goal of the Block Directory is to provide a safe place for WordPress users to find and install new blocks.
+
+<h3>User Expectations</h3>
+
+* Users will have a place they can go, both within their WordPress admin and on WordPress.org, where they can download and install blocks that have been pre-vetted for major security issues. 
+* Users will be able to one-click install blocks from their admin, one at a time.
+* Blocks will appear in their block library after installation and activation.
+
+<h3>Developer Expectations</h3>
+
+* Developers will have a concrete set of requirements and guidelines to follow when writing blocks for the Block Directory.
+* Following these guidelines will help ensure that the blocks they develop can be seamlessly installed in the editor.
+
+<h3>Definitions</h3>
+
+First up, some clarity. For the purposes of the Block Directory, we distinguish between two types of plugins:
+1. Plugins that exist solely to distribute a block.
+1. All other plugins, including those that bundle many independent blocks, plugins that contain blocks in addition to other functionality, and plugins with no blocks at all.
+
+These guidelines apply specifically to the first type of plugin, which we’ll call here a Block Plugin. If your plugin is of the second type, then the further guidelines and restrictions do not apply to your work. All plugins, be they block-only or not, are required to comply with the Detailed Plugin Guidelines https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/.
+
+<h3>Block Plugins and the Block Directory</h3>
+
+The Block Directory contains only Block Plugins, that is to say plugins that contain only a single independent block and a minimum of supporting code. Block plugins have the following characteristics:
+
+1. A specific type of WordPress plugin, with the same structure including a readme.txt file.
+1. Containing only blocks (typically one).
+1. Contain a minimum of server-side code. 
+1. Do not have any UI outside of the post editor.
+
+In addition to following the guidelines that apply to all WordPress plugins, Block Plugins that are submitted to the Block Directory must adhere to these guidelines:
+
+<h3>Guidelines</h3>
+
+<h4>1. Block Plugins are for the Block Editor.</h4>
+
+Your block plugins should contain a block, and a minimum of other supporting code. It should not contain any UX outside of the editor, such as WordPress options or `wp-admin` menus. Server-side code should be kept to a minimum.
```

Comment:
"Your block plugins must contain a block, and a minimum of other supporting code. It may not contain any UX outside of the editor, such as WordPress options or `wp-admin` menus. Server-side code should be kept to a minimum."

Stronger words here. Since we want blocks to 'just work' (no APIs no registration etc) we need to be firm here. No ~capes~ settings pages.